index.js

```diff
@@ -0,0 +1,47 @@
+const actionModel = require('./data/models/actionModel');
+const projectModel = require('./data/models/projectModel');
+const express = require('express');
+const router = express.Router(); 
+// const {check, validationResult } = require('express-validator/check');
+const port = 3333;
+
+//server def and init
+const server = express();
+server.use(express.json());
+
+server.get('/project/:id', async (req,res) => {
+    try {
+        let id = parseInt(req.params.id);
+        let result = await projectModel.findProjectbyId(parseInt(id));
+        const actions = await actionModel.findActionsbyProjectId(id);
+        res.status(200).json({...result, actions : actions});
```

Comment:
Nice use of the spread operator to insert the new key/value pairs into the object, also with async/await it looks so much cleaner, great job here Jonathan!